docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
+1. To open or create a file, use the `openFile()` method of the `FileSystemManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../api/latest/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../api/latest/device_api/tv/tizen/filesystem.html#FileSystemManager) applications).
 
-   The method returns a `FileStream` object, which is a handle to the opened file.
+   The method returns a `FileHandle` object, which is a handle to the opened or newly created file.
+
+   > **Note**  
+   > openFile() method used with mode 'w' creates new file or opens existing file and erases it's contents. Any data contained in a file opened in that mode will be lost.
+
+   Perform all actual operations, such as reading, writing, or closing on the file through the `FileHandle` object based on a current position, which can be obtained or modified through calling `seek()` operation on `FileHandle` object.
 
    ```
-   var documentsDir;
+   /* Opening file for write - file is created if not exists, */
+   /* otherwise existing file is truncated */
+   var fileHandleWrite = tizen.filesystem.openFile('documents/file', 'w');
+   console.log('File opened for writing');
+   ```
 
-   tizen.filesystem.resolve('documents', function(result) {
-       documentsDir = result;
-   });
+   - use writeString() or writeStringNonBlocking() to write `DOMString` to file.
+   ```
+   var stringToWrite = 'example string';
+   fileHandleWrite.writeString(stringToWrite);
+   ```
 
-   var testFile = documentsDir.createFile('test.txt');
-   if (testFile != null) {
-       testFile.openStream('rw', onOpenSuccess, null, 'UTF-8');
-   }
+   - use writeData() or writeDataNonBlocking() to write `Uint8Array` object contents to file.
+   ```
+   var arrayToWrite = new Uint8Array([11,22,88,99]);
+   fileHandleWrite.writeData(arrayToWrite);
    ```
 
-2. Perform all actual operations, such as reading, writing, or closing, on the file through the `FileStream` object based on a position attribute, which represents the current position in the file:`function onOpenSuccess(fs) {    /* Write HelloWorld to the file */    fs.write('HelloWorld');    /* Move pointer to the beginning */    fs.position = 0;    /* Read the file content from the beginning */    fs.read(testFile.fileSize);    /* Close the file */    fs.close();}`
+   - use writeBlob() or writeBlobNonBlocking() to write `Blob` object contents to file.
+   ```
+   var blobToWrite = new Blob(['example blob content']);
+   fileHandleWrite.writeBlob(blobToWrite);
+   ```
 
-3. To copy a file or directory, use the `copyTo()` method. The following example copies the files to the `images/backup/` directory. Since the third parameter is set to `true`, any existing files with the same name in the target directory are overwritten.
+   - when file is no longer needed, remember to close it
```

Comment:
Please rephrase to:

Close the file when it is not in use.

Reply:
Done.